content/cloud/faq.md

```diff
@@ -78,9 +94,6 @@ To learn more about what feature support looks like for various programming lang
 11:08:29 : Hello from Rust - updated
 ```
 
-- **How do I report a security concern, or concerns, with content hosted on the Fermyon Cloud?**
```

Comment:
I moved it over to the Pricing and Billing FAQ under ####Support

Reply:
Good point. I think they're most likely to look at [How To Get Help](https://developer.fermyon.com/cloud/support). Do you think it would be better to add it there / consolidate any support specific questions under that article?

Reply:
Good call.  "How to Get Help" seems like the ideal home for this (it already has a Report Abuse section).  Consolidating support questions there... not sure, it might get too baroque and hard to find the big red "bad shit is happening" button.  The "support" section in this PR includes things like "does this affect me using open-source Spin" and "where can I get a list of data sub-processors" which we definitely don't want cluttering up the core "I just need help" page.

Reply:
Agreed! I think in the future the Support page will get flushed out more.  As a fast follow to billing launch, I suspect we'll be prioritizing the creation of an outage hub as well

Reply:
@itowlson - friendly checkin to see if we can resolve this comment?

Reply:
Yep, looks good - thanks!